tests/test_worker.py

```diff
@@ -626,7 +626,7 @@ class TimeoutTestCase:
     def setUp(self):
         # we want tests to fail if signal are ignored and the work remain
         # running, so set a signal to kill them after X seconds
-        self.killtimeout = 10
+        self.killtimeout = 15
```

Comment:
FYI this was the only way I was able to fix a slow subprocess call in the pypy env on Travis CI.